src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -89,12 +90,16 @@
     (some #(= % "required") (:validators field-descriptor))
     (validator/validate "required" value)))
 
+(defn- add-link-target-prop
+  [text state]
+  [(string/replace text #"<a href=([^>]+)>" "<a target=\"_blank\" href=$1>") state])
```

Comment:
This depends on the exact format of HTML that `markdown-clj` outputs, but not much we can do about that. I did check that we could theoretically completely replace the link transformer, which would be a bit cleaner, but then I looked how complex it is and yeah no.